sopel/modules/clock.py

```diff
@@ -33,11 +34,7 @@ def f_time(bot, trigger):
     it will complain. If nothing can be found, it'll complain that the argument
     is not a valid timezone.
 
-    .. seealso::
-
-        Function :func:`~sopel.tools.time.format_time` is used to format
-        the current datetime according to the timezone (if found).
-
+    The format used can be set with the ``.settf`` or ``.setctf`` commands.
```

Comment:
So, in a PR to add missing help prefixes, we're going to create places where the help prefix can't be used? AFAIK there's no way to replace these `.`s with the correct prefix at runtime, unless you've got some fancy trick up your sleeve in that sopel-help plugin. 😛

Reply:
Hm. Maybe we should just not use any prefix for command name? After all, user should understand what it means, right?

Reply:
Should, yes. Doesn't mean we shouldn't try to develop some way of marking up a command name in command docstrings so the bot can add the prefix, but that sounds like a future feature thing.

Reply:
`*scream in refactoring*`